### DISCUSSION TITLE
Using a modifier on the peer class

### DISCUSSION THREAD

**Discussion by Ario99** [tailwindlabs/tailwindcss#15076] [Help] [answered]
So I noticed I can't do this. For example, you can't use peer in this way. `max-2xl:peer` I was wanting to know if that is intentional or is a limitation in tailwind css atm. I have a custom modifier that I would like to drive an animation. So if I could simply nullify the peer in this instance. It would allow me to use the currently existing peer class to drive this animation as it already does.

This is what I'm working with for my use case.

```
<button className="{custom-modifer}:peer ..."
>
  Account
</button>
<div className="... transition-transform scale-y-0 peer-focus-within:scale-y-100" />
```

**Comment by wongjn** [comment, marked as answer]
You'd add the modifier to the `peer-focus-within:` like:

```html
<button className="peer …">
  Account
</button>
<div className="… transition-transform scale-y-0 max-2xl:peer-focus-within:scale-y-100" />
```

**Reply by Ario99** [reply]
Thanks, this is a fairly simple solution.
